etc/disable-devel.inc

```diff
@@ -6,76 +6,53 @@ include /etc/firejail/disable-devel.local
 
 # GCC
 #blacklist /usr/lib/gcc	- seems to create problems on Gentoo
-blacklist /usr/bin/as
-blacklist /usr/bin/c++*
-blacklist /usr/bin/c8*
-blacklist /usr/bin/c9*
-blacklist /usr/bin/cpp*
-blacklist /usr/bin/g++*
-blacklist /usr/bin/gcc*
-blacklist /usr/bin/gdb
-blacklist /usr/bin/ld
-blacklist /usr/bin/x86_64-unknown-linux-gnu-gcc*
-blacklist /usr/bin/x86_64-linux-gnu-g++*
-blacklist /usr/bin/x86_64-linux-gnu-gcc*
-blacklist /usr/bin/x86_64-unknown-linux-gnu-g++*
+blacklist ${PATH}/as
+blacklist ${PATH}/c++*
```

Comment:
I just noticed that `cc` has not yet been blacklisted. Might be a useful addition here (as `c++` is also there).